02.calendar/cal.js

```diff
@@ -0,0 +1,59 @@
+const args = require('minimist')(process.argv.slice(2)) // { _: [], y: 2022, m: 9 }
+const year = args.y // コマンド引数で年を取得
+const month = args.m // コマンド引数で月を取得
+const weekdays = ['日', '月', '火', '水', '木', '金', '土'] // 曜日を初期化
+const startDate = new Date(year, month - 1) // 月初の日付を取得
+const startWday = startDate.getDay() // 月初の曜日を取得
+const lastDate = new Date(new Date(year, month, 0)) // 月末の日付を取得
+const oneSpace = ' ' // 半角空白を初期化
+let firstSaturday // 最初の土曜日の日付を宣言
+
+// ========コマンド引数で取得した年月を出力========
+console.log((oneSpace * 5) + year + '年' + 'month')
```

Comment:
JSにもRubyのstring interpolation(式展開)のような、template literalの構文があります。

```suggestion
console.log(`${oneSpace * 5}${year}年${month}`)
```

https://developer.mozilla.org/ja/docs/Web/JavaScript/Reference/Template_literals

Reply:
@yoshitsugu 

テンプレートリテラルを活用する形式に修正しました。